<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>RE Digestion</title>
  <meta name="description" content="Portfolio website for Tankiso Phoofolo — SIEM, Splunk dashboards, log analysis." />
  <link rel="stylesheet" href="styles.css" />
</head>
<body>
  <header>
    <!--<h1>Tankiso Phoofolo</h1>-->
    <p><h2>RE Digestion</h2></p>
    <nav>
      <a href="index.html">Home</a>
      <!--<a href="#contact">Contact</a>-->
    </nav>
  </header>

  
  <section class="standaloneIMG">
     <img src="Images/Restriction Digestion_enzymes.jpg" alt="RE Digestion enzymes">
  </section>
</br>
<div class="alignwell">
<h4>Restriction Enzyme Digestion and Sequencing</h4>

  <p>The purified plasmid DNA was digested using restriction endonucleases as shown in table 1 below.
   Appendix A illustrates the binding sites of the enzymes to the light chain (LC) plasmid vector (VRCO3_LK_1_5) and the heavy chain (HC) plasmid vector (VRC03-HC-1_5). After digestion, the mixture was incubated for 1.5 hours at 37°C. 
   An undigested plasmid was mixed with a 6X Loading Dye (BioLabs, USA) to be used as a control. After incubation, the restriction mixture was run on a 1.3% gel. The undigested plasmid and a 10 000 bp GeneRuler mix DNA ladder (Thermo Scientific, Illinois, USA) were also run alongside the restriction products on the gel for 
   60 minutes at 90 volts (V) in 1X TAE buffer using the PowerPac™ Basic Power Supply (BioRad, USA). The gel was then immersed in a 10 000X GelRed (Biotium, USA) solution prepared according to table 2 and put on a shaker (70 rpm) for 45 minutes to post-stain. 
   Thereafter, the band sizes were confirmed using GelDoc™ XR+ and Image Lab™ Software (BioRad, USA).
  </p>
 
  <h6>Table 1: Components and volumes for Plasmid DNA restriction reaction</h6>
  <table>
    <tr>
      <th>Component</th>
      <th>Volume (μl)</th>
      
    </tr>
    <tr>
      <td>Plasmid DNA</td>
      <td>2 (=1μg)</td>
    </tr>

    <tr>
      <td>10X FastDigest Green Buffer</td>
      <td>5</td>
    </tr>
    
    <tr>
      <td>Agel (BshTI) (for HC and LC)</td>
      <td>1</td>
    </tr>

    <tr>
      <td>SalI FastDigest enzyme (for HC) / BsiWI (PFI23II) FastDigest enzyme (for LC)</td>
      <td>1</td>
    </tr>

    <tr>
      <td>Nuclease-free water</td>
      <td>to 50 μl</td>
    </tr>
    
    </td>
  </table>

    <h6>Table 2: Post-stain GelRed preparation</h6>
    
  <table>
    <tr>
      <th>Component</th>
      <th>Volume (ml)</th>
      
    </tr>
    <tr>
      <td>GelRed™ Nucleic Acid gel Stain</td>
      <td>0.01</td>
    </tr>

    <tr>
      <td>0.1M NaCl</td>
      <td>45</td>
    </tr>
    
    <tr>
      <td>Distilled water</td>
      <td>5</td>
    </tr>

      
    
    </td>
    </table>

<br>
<h4>Sanger Sequencing</h4>
<p>Sanger sequencing was performed to further confirm successful transformation of the recombinant LC and HC plasmid DNA using the primers shown in table 3. 
  The reagents for the sequencing reaction are shown in table 4.</p>

  <h6>Table 3: Primer sequences </h6>
  
  <table>
    <tr>
      <th>Primer</th>
      <th>Forward/reverse</th>
      <th>Nucleotide sequence (5’ to 3’)</th>
      <th>Tm (°C)</th>
      <th>%GC</th>
      
    </tr>
    <tr>
      <td>VRC03-Seq-F</td>
      <td>Forward</td>
      <td>CTAGTTAACGGTGGAGGGCAGTGT</td>
      <td>59.1</td>
      <td>54</td>
    </tr>

    <tr>
      <td>VRC03-Seq-R</td>
      <td>Reverse</td>
      <td>CACCCCCCAGAATAGAATGACACC</td>
      <td>59.1 </td>
      <td>54</td>
    </tr>
    
    <tr>
      <td>1012-F</td>
      <td>Forward</td>
      <td>GCTGACAGACTAACAGACTG</td>
      <td>51.8</td>
      <td>50</td>
    </tr>

    <tr>
      <td>1012-R</td>
      <td>Reverse</td>
      <td>GCAAACAACAGATGGCTGGC</td>
      <td>53.8</td>
      <td>55</td>
    </tr>
   
  </table>

  <h6>Table 4: Sequencing reaction components</h6>
    
  <table>
    <tr>
      <th>Reagents </th>
      <th>Volume (μl)</th>
      
    </tr>
    <tr>
      <td>Terminatory Ready Reaction Mix </td>
      <td>1</td>
    </tr>

    <tr>
      <td>5X Sequencing Buffer</td>
      <td>2</td>
    </tr>
    
    <tr>
      <td>Primer (2 pmol/μl)</td>
      <td>3</td>
    </tr>
    
    <tr>
      <td>Template DNA</td>
      <td>2 (=1 μg)</td>
    </tr>

    <tr>
      <td>Nuclease-free water</td>
      <td>2</td>
    </tr>

     <tr>
      <td><strong>Total</strong></td>
      <td><strong>10</strong></td>
    </tr>
        
  </table>

  <p>The routine PCR for sequencing was performed using the ProFlex™ PCR system (Applied Biosystems, 
    New York, USA) thermal cycler set up as shown in table 5.</p>

  <h6>Table 5: PCR thermocycling conditions</h6>
  <table>
    <tr>
      <th>STEP </th>
      <th>TEMPERATURE (°C)</th>
      <th>TIME </th>
      
    </tr>
    <tr>
      <td>Initial Denaturation  </td>
      <td>96</td>
      <td>1 minute</td>
    </tr>

    <tr>
      <td>25 cycles:  Denaturation </td>
      <td>96</td>
      <td>10 seconds</td>
    </tr>
    
    <tr>
      <td>Annealing</td>
      <td>50</td>
      <td>5 seconds</td>
    </tr>
    
    <tr>
      <td>Extension</td>
      <td>60</td>
      <td>4 minutes</td>
    </tr>

    <tr>
      <td>Hold </td>
      <td>4</td>
      <td>∞</td>
    </tr>
          
  </table>

  <p>Ethanol/EDTA precipitation was used for the post-reaction cleanup. 5 μl of 125 mM EDTA and 60 μl of absolute ethanol were added to eight 1.5 ml microcentrifuge tubes. Each sequence reaction mixture was diluted to 20 μl by adding 10 μl nuclease-free water. 
    Each diluted sequence reaction mixture was then added to an EDTA and absolute ethanol mixture. The tubes were vortexed for 60 seconds. The tubes were then left at room temperature for 15 minutes to allow for DNA to precipitate. After that, the tubes were centrifuged at 14000 x g for 20 minutes at 4°C. 
    The supernatant was completely removed without disturbing the pellet. In each tube, 500 μl of 70% ice-cold ethanol was added, and they were centrifuged at 14 000 x g for 10 minutes at 4°C. The supernatant was removed, and the tubes were incubated at 37°C with open lids until completely dry. 
    Thereafter, the tubes were stored at 4°C in the dark. The samples were submitted to the Department of Genetics at the University of the Free State for sequencing.</p>


<h5>Key skills applied & strengthened:</h5>
<p>- Setting up and performing restriction enzyme digestion reactions</br>
- Gel electrophoresis for DNA fragment analysis</br>
- Post-staining and visualization of DNA fragments using GelRed</br>
- Interpretation of gel electrophoresis results for plasmid verification</br>
- Post-reaction cleanup using ethanol/EDTA precipitation</br>
- Understanding and applying PCR thermocycling conditions for sequencing</br>
- Collaboration with sequencing facilities for data acquisition
</p>


</div>
  

  <footer>
    <p>Tankiso Phoofolo - Portfolio</p>
  </footer>
</body>
</html>
